shepherd/Utils.py

```diff
@@ -1,58 +1,99 @@
 # pylint: disable=invalid-name
 class SHEPHERD_HEADER():
     START_NEXT_STAGE = "start_next_stage"
+        # START_NEXT_STAGE{}: starts the next stage
+
     RESET_CURRENT_STAGE = "reset_current_stage"
+        # RESET_CURRENT_STAGE{}: resets the current stage
```

Comment:
Consolidate these comments in a docstring. Up to you to pick the docstring format, but I recommend [Google Python style](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) or [RST](http://docutils.sourceforge.net/rst.html).